dependencies.yaml

```diff
@@ -466,6 +466,7 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - dask-ml
+          - hdbscan==0.8.38
```

Comment:
Since there are `post` releases in some cases, think we need to relax this at least a little bit

```suggestion
          - hdbscan>=0.8.38,<0.8.39
```